.github/workflows/my-ci-pipeline.yml

```diff
@@ -29,4 +29,5 @@ jobs:
           password: ${{ secrets.DOCKER_API_KEY }}
    - uses: actions/checkout@v2 
    - run: docker build --target production --tag aexalt/todo-app:latest .
-   - run: docker push aexalt/todo-app:latest
\ No newline at end of file
+   - run: docker push aexalt/todo-app:latest
+   - run: curl -dH -X POST ${{ secrets.AZURE_WEBHOOK }} 
```

Comment:
One nice addition here can be adding the [--fail](https://curl.se/docs/manpage.html#-f) flag, as otherwise curl reports success as long as a successful connection was made, even if the response reports a failure (e.g. a status code > 400) which can lead to confusing pipeline results